build/stf-collect-logs/tasks/main.yml

```diff
@@ -1,35 +1,23 @@
 ---
-- name: "Get builds"
-  ansible.builtin.shell:
-    cmd: |
-      echo "*** [INFO] Showing oc get builds" > {{ logfile_dir }}/post_oc_get_builds.log 2>&1
-      oc -n {{ namespace }} get builds -oyaml >> {{ logfile_dir }}/post_oc_get_builds.log 2>&1
-      echo "*** [INFO] Showing oc get builds -oyaml" >> {{ logfile_dir }}/post_oc_get_builds.log 2>&1
-      oc -n {{ namespace }} get builds -oyaml >> {{ logfile_dir }}/post_oc_get_builds.log 2>&1
-      cat {{ logfile_dir }}/post_oc_get_builds.log
+- name: "Get resources logs [ Builds, Subscriptions, Image, Imagestreams, Pods ]"
+  ansible.builtin.shell: |
+    for resource in {{ resource_types|join(' ') }}; do
+      log_file="{{ logfile_dir }}/post_oc_get_$resource.log"
+      echo "*** [INFO] Showing oc get '$resource'" > "$log_file" 2>&1
+      oc -n {{ namespace }} get "$resource" >> "$log_file" 2>&1
+      echo "[INFO] oc get '$resource' -oyaml" >> "$log_file" 2>&1
+      oc -n {{ namespace }} get "$resource" -oyaml >> "$log_file" 2>&1
+    done
+  delay: 10
   ignore_errors: true
   changed_when: false
 
-- name: "Get subscription details"
+- name: "Get Additional Information  details"
   ansible.builtin.shell:
     cmd: |
-      oc -n {{ namespace }} get subscriptions > {{ logfile_dir }}/post_oc_get_subscriptions.log 2>&1
       oc -n {{ namespace }} describe subscription service-telemetry-operator >> {{ logfile_dir }}/post_oc_get_subscriptions.log 2>&1
```

Comment:
While you're here....
The contents of this file would be clearer if it was called "post_oc_describe_subscription_STO.log", which would match the format of the names of the other log files.

It is currently called ``post_oc_get_subscriptions.log`` because the first command was ``oc get subscriptions``